test/test-utils.js

```diff
@@ -0,0 +1,90 @@
+/*jshint expr: true*/
+"use strict";
+var chai = require("chai");
+var expect = chai.expect;
+chai.config.includeStack = true;
+var utils = require("../utils");
+
+describe("Utilities testing", function() {
+  it("checks the integrity of the utilities", function() {
+    expect(utils).to.be.an('object');
+    expect(utils).to.have.all.keys([
+      'isValidDirectory', 'isValidFile',
+      'readFile', 'readJsonFile',
+      'defaults', 'normalizeApiPath'
+    ]);
+  });
+
+  describe("File verification tests", function() {
```

Comment:
Yes, because it describes the functionality of that function. If you want to be fancy you can add a period in front of it :) Some people do `#isValidFile` or `.isValidFile`